tests/test_middleware_interface.py

```diff
@@ -85,10 +85,9 @@ def setUp(self):
                                 writeable=False)
         instrument = "lsst.obs.decam.DarkEnergyCamera"
         self.input_data = os.path.join(data_dir, "input_data")
-        self.repo = tempfile.TemporaryDirectory()
-        instrument = lsst.obs.base.utils.getInstrument(instrument)
-        butler = Butler(Butler.makeRepo(self.repo.name), instrument="LSSTCam", writeable=True)
-        self.interface = MiddlewareInterface(central_butler, self.input_data, instrument, butler,
+        repo = tempfile.TemporaryDirectory()
+        self.butler = Butler(Butler.makeRepo(repo.name), writeable=True)
```

Comment:
`self.local_butler`, just to be unambiguous?

Reply:
It ends up making a bunch of things rather longer though: we use it quite a bit.